modules/ROOT/pages/salesforce/salesforce-connector-100-config-topics.adoc

```diff
@@ -266,33 +256,21 @@ The JSON received as a response from the push event operation looks something li
 [[batchdata]]
 == Load Data in Batches
 
-The Salesforce Bulk API loads batches of your organization's data into Salesforce.
-
-The Salesforce connector provides the Create and Create Bulk operations for working
-with the Bulk API.
-
-For all bulk operations, Salesforce handles the creation process in the background, so the connector does not reply with a collection of SaveResults because it does not have them yet. Instead, the connector replies with a BatchInfo object, which contains the ID of the batch and the ID of the job it creates to upload those objects.
-
-[[trackbulk]]
-=== Track Bulk Data Status
+The Salesforce Bulk API loads batches of your organization's data into Salesforce. Salesforce Connector provides the Create and Create Bulk operations for working with the Bulk API.
 
-You can monitor a Bulk API batch in Salesforce through the Job ID for the Bulk Data Load Jobs.
+For all bulk operations, Salesforce handles the creation process in the background. The connector replies with a BatchInfo object, which contains the ID of the batch and the ID of the job it creates to upload the batched objects.
 
-The job detail page in Salesforce includes a related list of all the batches for the job. The related list provides View Request and View Response links for each batch. If a batch is a CSV file, the links return the request or response in CSV format. If a batch is an XML file, the links return the request or response in XML format.
-
-In Salesforce, you can track the status of bulk data load jobs and their associated batches:
+In Salesforce, you can track the status of bulk data load jobs and their associated batches by using the job ID for the Bulk Data Load Jobs:
 
 . Click YOUR_NAME > *Setup* > *Monitoring* > *Bulk Data Load Jobs*.
 . Click the job ID to view the job detail page.
 
-The job detail page includes a related list of all the batches for the job. The related list provides *View Request* and *View Response* links for each batch. If the batch is a CSV file, the links return the request or response in CSV format. If the batch is an XML file, the links return the request or response in XML format.
+The job detail page includes a related list of all the batches for the job. The related list provides `View Request` and `View Response` links for each batch. If the batch is a CSV file, the links return the request or response in CSV format. If the batch is an XML file, the links return the request or response in XML format.
```

Comment:
What is "a related list"? Do we mean "The job detail page includes a list of all the batches related to the job."?